os_net_config/cli.py

```diff
@@ -574,5 +629,33 @@ def get_iface_config(
     return iface_array
 
 
+def minimum_config(provider,
```

Comment:
We will need a --no-network-config option for --purge-provider for applying min config without trying to apply the network_config